packages/lesswrong/lib/collections/forumEvents/schema.ts

```diff
@@ -59,6 +67,38 @@ const schema: SchemaType<"ForumEvents"> = {
     type: String,
     control: "ImageUpload",
   },
+  includesPoll: {
+    ...defaultProps(),
+    optional: true,
+    nullable: true,
+    type: Boolean,
+    control: "FormComponentCheckbox",
+  },
+  /**
+  Used to store public event data, like public poll votes.
+  For the AI Welfare Debate Week, it was structured like:
+  {<userId>: {
+    x: <number>,
+    points: {
+      <postId>: <number>
+    }
+  }}
+   */
+  publicData: {
+    type: Object,
+    blackbox: true,
+    optional: true,
+    nullable: true,
+    hidden: true,
+    canRead: ["guests"],
+    canCreate: ["members"],
+  },
```

Comment:
Not super happy about this one, but I figured we might want to accommodate events similar to the good news banner, where there is a different sort of publicly visible event data 🤷‍♀️